routers/repo/issue.go

```diff
@@ -453,6 +454,8 @@ func NewIssuePost(ctx *context.Context, form auth.CreateIssueForm) {
 		return
 	}
 
+	notification.Service.NotifyIssue(issue)
```

Comment:
Should this be put in background too ?
is: `go notification.Service.NotifyIssue(issues)`

If I read the code correctly the NotifyIssue writes into a channel, is that operation immediate or blocks until someone *reads* from the channel ? (lame gopher here)